client/gradle/openapi.gradle

```diff
@@ -6,6 +6,7 @@ swaggerSources {
   client {
     inputFile = file(tasks.getByPath(':openapi:buildOpenApi').openApiSource)
     code {
+      jvmArgs = ['--add-opens=java.base/java.util=ALL-UNNAMED'] // for Java 16+
```

Comment:
Is this for module system compatibility?

Reply:
This is part of the workaround because the codegen doesn't follow the Java17 access rules.